### DISCUSSION TITLE
You Can Now Properly Hide Steam Games by Marking Them as Private

### DISCUSSION THREAD

**Discussion by visionarylab** [SteamRE/SteamKit#1377] [General]
### What problem is this feature trying to solve?

Any API to mark one or more Steam Games Private

### How would you like it to be solved?

Any Existing API can mark steam games to private?

### Have you considered any alternative solutions

No, API access is better.

### Additional Information

You Can Now Properly Hide Steam Games by Marking Them as Private. 

- It would be better to mass hide the game via API

**Comment by abdou2a** [comment]
did u find out how to mass hide steam games ?
